bundles/pom.xml

```diff
@@ -66,7 +66,7 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>rat-maven-plugin</artifactId>
-                <version>1.0-alpha-2</version>
+                <version>1.0-alpha-3</version>
```

Comment:
Ups... this is wrong, we are using `apache-parent:21` and the rat plugin is defined there are follow:
```
        <plugin>
          <groupId>org.apache.rat</groupId>
          <artifactId>apache-rat-plugin</artifactId>
          <version>0.12</version>
        </plugin>
```

So it would be better to just redefine this section to:
```
        <plugin>
          <groupId>org.apache.rat</groupId>
          <artifactId>apache-rat-plugin</artifactId>
          ...
```

The same for the pom in the sitegraph plugin

Reply:
Oh, I see. the "zero-versions" seem to be newer than 1.x. ;-)

I try it out later.

see http://creadur.apache.org/rat/apache-rat-plugin/changes-report.html

Reply:
@lukaszlenart I removed the explicit version for `apache-rat-plugin`.